src/pages/kb/data-sources/azure-sql-database.md

```diff
@@ -1,10 +1,14 @@
 ---
 category: querying
 parent_category: data-sources
-title: Microsoft Azure SQL Database
+title: Microsoft Azure SQL Database & Synapse
 slug: azure-sql-database
 ---
 
-When setting up a connection to Azure SQL Database, you need to pick "Microsoft SQL Server" as the type.
+Redash has two SQL Server connectors which use different python libraries: **Microsoft SQL Server** (`pymssql`) and **Microsoft SQL Server (ODBC)** (`pyodbc`). Either connector will work with Azure SQL Database. The ODBC connector is required to connect with Synapse.
```

Comment:
Let's instruct users to use the ODBC one for new connections. The old one will be removed eventually as it has no benefit.

Reply:
Done 9eb0622ff655d21be3986540be4796111b5281eb

Reply:
Updated my screenshot in the PR description.